setup.py

```diff
@@ -19,6 +21,8 @@
     'psutil==2.1.3',
     'hypothesis',
     'pgpy',
+    'tox==2.3.1',
+    'coverage==4.0.3',
```

Comment:
Should this be added as optional packages? (e.g. pip install taskcluster-client[tests])
Otherwise, users of this package would be forced to get these two packages installed.